### ISSUE TITLE
Write user documentation

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#29]
I believe we can create a `/doc/user` folder with markdown documents stating how to use TMail:

 - [ ] Configuration of your account
 
(Login, password, URL, what do they mean, how URL session detection works, how the authentication flow works etc...)

 - [ ] Explanation of the display and basic UI tasks (with screenshots!)
     - Explanations of the setting page.
     - How do I switch of mailbox?
     - Which actions can I perform... with mailboxes? With emails? Screenshots...
     
This would serve 2 goals:

 - Help out newbie users that might be lost...
 - Serve as a portfolio to show how the apps looks like and convince people of using it!
 
 Basically this task can be done by ANYONE in the emailing team. I expect **BA** and **backend DEVs** to contribute to it. (With the side goal of having *as much people as possible contributing to this repository*)
 
  - [ ] Link the user doc to the readme
  - [ ] see if we can host the doc as a github content page

**Comment by chibenwa** [comment]
Duplicate
